### ISSUE TITLE
Xamarin.Forms.ActivityIndicator in Forms v 2.5 doesn't work (UWP)

### ISSUE THREAD

**Issue by jasells** [xamarin/Xamarin.Forms#1483]
Bug report best practices: https://github.com/xamarin/Xamarin.Forms/wiki/Submitting-Issues

### Description
Activity indicator was working in previous version (Forms v 2.4.x) on UWP.  Updating to Forms v. 2.5 broke it.  Android still works as expected.

### Steps to Reproduce

1. 
2. 
3. 

### Expected Behavior
ActivityIndicator should be visible and animated while IsRunning and IsVisible properties are True, and change invisible when false.

### Actual Behavior
The area of the view is a grey box, the animation is (barely) visible only once.

### Basic Information

- Version with issue:  Xamarinf.Forms v 2.5.x
- Last known good version: 2.4.0.91020
- IDE:  VS2017
- Platform Target Frameworks: 
  - UWP:  10586 -15063 

### Screenshots

Wrong visual:
![activitynowork](https://user-images.githubusercontent.com/6353954/34472300-81250f7a-ef24-11e7-9908-0a824c2ab489.PNG)

Correct:
![activitywork](https://user-images.githubusercontent.com/6353954/34472354-16eebf96-ef26-11e7-90a6-572c73064782.PNG)

**Comment by samhouts** [comment]
[Issue1483.zip](https://github.com/xamarin/Xamarin.Forms/files/1598516/Issue1483.zip)

I'm unable to reproduce this. I have attached a test project. Can you please adjust it so that it exhibits the issue? Thanks.

**Comment by jasells** [comment]
[Issue1483.zip](https://github.com/xamarin/Xamarin.Forms/files/1598918/Issue1483.zip)

I cannot repro the issue in your solution either.

I zipped my (original) solution that shows the issue, when Xamarin.Forms is updated on the UWP project.  As packaged, UWP is referencing Forms v2.4, and it works as expected.  To see the issue, update Xamarin.Forms nugget package on UWP project.

It will loop toggling the ActivityIndicator every ~3.5 sec.

**Comment by kingces95** [comment]
Activity animation degraded after update using customer reproduction.

**Comment by joseluisct** [comment]
I have this issue also, it happens after upgrade to Xamarin Forms 2.5

I've been testing the conditions of this to happen and it happens when:
· The page is a NavigationPage.

But if it's a ContentPage it works OK, even putting the NavigationPage as Detail of MasterDetail, it works OK.

**Comment by maxvodoo** [comment]
same issue

**Comment by eploginov** [comment]
same issue, for new project on Xamarin.Forms 2.5.*

**Comment by bojan96** [comment]
Have a same issue

**Comment by acastr7** [comment]
This might be duplicate of #1399. There is also a Bugzilla report here https://bugzilla.xamarin.com/show_bug.cgi?id=43285

**Comment by mgwalm** [comment]
This is still a problem in forms version 3.0.0.482510.
As stated above it works on a contentpage, but not when one uses a NavigationPage.

I've compared the UWP Visual tree of an app that is using a ContentPage to one that is using a NavigationPage, and on the ContentPage there is a DeterminateRoot (border) as the last child of the FormsProgressBar that has an Animation and Visual State property (ie the storyboard is running). However on the NavigationPage version the  DeterminateRoot (border) is present but the Animation and Visual State property is missing, ie, the storyboard is not running. Leave a comment

**Comment by Matti-Koopa** [comment]
This sounds like it can be fixed with a custom Renderer. Are you able to share your code, @mgwalm?
